aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/data-partitioning-strategies.md

```diff
@@ -1,8 +1,8 @@
 ---
 uid: aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/data-partitioning-strategies
 title: "Data Partitioning Strategies (Building Real-World Cloud Apps with Azure) | Microsoft Docs"
-author: MikeWasson
-description: "The Building Real World Cloud Apps with Azure e-book is based on a presentation developed by Scott Guthrie. It explains 13 patterns and practices that can he..."
+author: Rick-Anderson
+description: "There is an e-book called: Building Real World Cloud Apps with Azure. The e-book is based on a presentation developed by Scott Guthrie and explains 13 patterns and practices that can he..."
```

Comment:
Remove quotes. Suggest we go with "The e-book Building Real World Cloud Apps with Azure is based on. a presentation developed by Scott Guthrie...."